src/rpm/builder.rs

```diff
@@ -1023,34 +1045,6 @@ impl RPMBuilder {
         }
 
         let header = Header::from_entries(actual_records, IndexTag::RPMTAG_HEADERIMMUTABLE);
-
-        //those parts seem to break on fedora installations, but it does not seem to matter for centos.
-        // if it turns out that those parts are not really required, we will delete the following comments
-
-        // self.requires.push(Dependency::rpm_lib(
-        //     "rpmlib(VersionedDependencies)".to_string(),
-        //     "3.0.3-1".to_string(),
-        // ));
-
-        // self.requires.push(Dependency::rpm_lib(
-        //     "rpmlib(PayloadFilesHavePrefix)".to_string(),
-        //     "4.0-1".to_string(),
-        // ));
-
-        // self.requires.push(Dependency::rpm_lib(
-        //     "rpmlib(CompressedFileNames)".to_string(),
-        //     "3.0.4-1".to_string(),
-        // ));
-
-        // self.requires.push(Dependency::rpm_lib(
-        //     "rpmlib(PayloadIsXz)".to_string(),
```

Comment:
This might be the part that was breaking.  Obviously the payload is not XZ because we don't support it.